orchagent/orchdaemon.cpp

```diff
@@ -513,6 +517,19 @@ void OrchDaemon::start()
                 }
             }
         }
+        if (gIsSwitchShutdown) {
+            static time_t last_log = 0;
+
+            if ((time(NULL) - last_log) > SWITCH_SHUTDOWN_LOG_INTERVAL_IN_SECS) {
+                SWSS_LOG_ENTER();
```

Comment:
>SWSS_LOG_ENTER(); [](start = 16, length = 17)

LOG_ENTER is for function body. What is the purpose here?

Reply:
True. Dropped